python/testDeleteInstuckProgress.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+import nose.tools as nt
+import deleteInstuckProgress
```

Comment:
I would advise you to use `_` and lowecase everywhere instead of camelCase for Python stuff. That is the [preferred coding style](https://www.python.org/dev/peps/pep-0008/#package-and-module-names).